src/server/game/Battlegrounds/BattlegroundMgr.cpp

```diff
@@ -434,49 +500,12 @@ Battleground* BattlegroundMgr::CreateNewBattleground(BattlegroundTypeId bgTypeId
 
     Battleground* bg = NULL;
```

Comment:
its not in your PR but i think here nullptr be better to

Reply:
agree